<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UnTech::GuiQt::Resources::ResourceFileCentralWidget</class>
 <widget class="QWidget" name="UnTech::GuiQt::Resources::ResourceFileCentralWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>360</height>
   </rect>
  </property>
 </widget>
 <resources/>
 <connections/>
</ui>
